moveit_core/planning_interface/include/moveit/planning_interface/planning_interface.h

```diff
@@ -165,7 +165,7 @@ class PlannerManager
   /// It is assumed that motion plans will be computed for the robot described by \e model and that any exposed ROS
   /// functionality
   /// or required ROS parameters are namespaced by \e ns
-  virtual bool initialize(const robot_model::RobotModelConstPtr& model, const std::string& ns);
+  virtual bool initialize(const robot_model::RobotModelConstPtr& model, std::shared_ptr<rclcpp::Node>& node);
```

Comment:
Are you sure we will no longer need the namespace? Also, can you point me to some code where you use the node?